docs/maintainers-guide.md

```diff
@@ -1,87 +1,96 @@
+-   `develop` is the default branch, the target of active development, and should at all times
+    should be a viable candidate for the next release.
+-   `main` is a tightly-controlled branch that release engineering uses for final tags deployed to
+    production.
 
 ### Creating issues on GitHub
 
-GitHub's [issues](https://github.com/hashgraph/hedera-services/issues) are used as the primary method for tracking
-project changes.
-Any actionable item that need to be addressed, should be associated to an issue in GitHub.
+GitHub's [issues](https://github.com/hashgraph/hedera-services/issues) are used as the primary
+method for tracking project changes. Any actionable item that need to be addressed, should be
+associated to an issue in GitHub.
 
 There are three types of issues:
-- **Bug**: These track issues with the code
-- **Documentation**: These track problems or insufficient coverage with the documentation
-- **Enhancement**: These track specific feature requests and ideas until they are complete. This should only be for
-  trivial or minor enhancements. If the feature is sufficiently large, complex or requires coordination among multiple
-  Hedera projects, it should first go through the Hedera Improvement Proposal process.
 
-Any issue created should be added to [Services Sprint Tracking](https://github.com/orgs/hashgraph/projects/13) 
-(if it is targeted to be addressed in the current sprint)and the associated project type in `Projects` tab. It should 
-also have the targeted milestone set on it.
+-   **Bug**: These track issues with the code
+-   **Documentation**: These track problems or insufficient coverage with the documentation
+-   **Enhancement**: These track specific feature requests and ideas until they are complete. This
+    should only be for trivial or minor enhancements. If the feature is sufficiently large, complex
+    or requires coordination among multiple Hedera projects, it should first go through the Hedera
+    Improvement Proposal process.
 
-For example, a documentation change targeted for the current sprint in 0.30.0 release should be associated to
-[Services Sprint Tracking](https://github.com/orgs/hashgraph/projects/13) and [Documentation](https://github.com/hashgraph/hedera-services/projects/32#card-85521291)
-projects, with 0.30 milestone on it.
+Any issue created should be added to
+[Services Sprint Tracking](https://github.com/orgs/hashgraph/projects/13) (if it is targeted to be
+addressed in the current sprint)and the associated project type in `Projects` tab. It should also
+have the targeted milestone set on it.
+
+For example, a documentation change targeted for the current sprint in 0.30.0 release should be
+associated to [Services Sprint Tracking](https://github.com/orgs/hashgraph/projects/13) and
+[Documentation](https://github.com/hashgraph/hedera-services/projects/32#card-85521291) projects,
+with 0.30 milestone on it.
 
 <p>
     <img src="./assets/labels-on-issue.png"/>
```

Comment:
We should use markdown syntax for the image inclusion.
```
<p>
    <img src="./assets/labels-on-issue.png"/>
</p>
```

becomes:
`![gitflow-branching-model](./assets/labels-on-issue.png)`